org.eclipse.zest.layouts/src/org/eclipse/zest/layouts/algorithms/RadialLayoutAlgorithm.java

```diff
@@ -53,19 +52,23 @@ public RadialLayoutAlgorithm(int styles) {
 
 	@Override
 	public void setLayoutArea(double x, double y, double width, double height) {
-		throw new RuntimeException("Operation not implemented");
+		throw new RuntimeException("Operation not implemented"); //$NON-NLS-1$
 	}
 
 	@Override
 	protected boolean isValidConfiguration(boolean asynchronous, boolean continueous) {
-		if (asynchronous && continueous)
+		if (asynchronous && continueous) {
 			return false;
-		else if (asynchronous && !continueous)
+		}
+		if (asynchronous && !continueous) {
 			return true;
-		else if (!asynchronous && continueous)
+		}
+		if (!asynchronous && continueous) {
 			return false;
-		else if (!asynchronous && !continueous)
+		}
+		if (!asynchronous && !continueous) {
 			return true;
+		}
 
```

Comment:
```
if (asynchronous && continueous) {
	return false;
}
if (asynchronous && !continueous) {
	return true;
}
if (!asynchronous && continueous) {
	return false;
}
if (!asynchronous && !continueous) {
	return true;
}
return false;
```
=> `return !continueous`

Reply:
I found this code extra spooky and was to afraid to change it. But you are totally right. Changed it accordingly.